pkg/controller/middleware/template.go

```diff
@@ -33,14 +34,10 @@ func PopulateTemplateVariables(ctx context.Context, config *config.ServerConfig)
 			ctx := r.Context()
 
 			m := controller.TemplateMapFromContext(ctx)
-
-			// Set initial variables if they are not set.
-			if _, ok := m["server"]; !ok {
-				m["server"] = config.ServerName
-			}
-			if _, ok := m["title"]; !ok {
-				m["title"] = config.ServerName
-			}
+			m["server"] = config.ServerName
+			m["title"] = config.ServerName
```

Comment:
Don't you not want to overwrite this?

Reply:
It's actually not possible to overwrite them since we've moved this into the middleware. This is an artifact from whenever they were lazy-added, but now we force-add these attributes very early in the middelware.